public/blocks/furniture/examples/content-publishedinfo.html

```diff
@@ -14,6 +14,16 @@
   <link type="font/woff2" href="https://assets.nhs.uk/fonts/FrutigerLTW01-65Bold.woff2" rel="preload" as="font" crossorigin>
   <link href="/css/nhsuk-4.1.0.min.css" rel="stylesheet" />
   <link href="/css/hee.css" rel="stylesheet" />
+  <style>
```

Comment:
@dawntaroni shouldn't we be including this as SASS somewhere instead? Seems like it's replicated in other templates so probably a good idea, so that we don't have to change this in multiple places :slightly_smiling_face: